src/user/model.rs

```diff
@@ -0,0 +1,135 @@
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use sqlx::postgres::PgPool;
+use sqlx::postgres::PgRow;
+use sqlx::{FromRow, Row};
+
+// struct for receiving user input
+#[derive(Serialize, Deserialize)]
+pub struct UserRequest {
+    pub user_id: i32,
+    pub first_name: String,
+    pub last_name: String,
+}
+
+// struct to represent database record
+#[derive(Serialize, FromRow)]
+pub struct User {
+    pub user_id: i32,
+    pub first_name: String,
+    pub last_name: String,
+}
+
+// implementation of Actix Responder for User struct so we can return
+// a User upon call to handler
+// impl Responder for User {
+//     type Error = Error;
+//     type Future = Ready<Result<HttpResponse, Error>>;
+//
+//     fn respond_to(self, _req: &HttpRequest) -> Self::Future {
+//         let body = serde_json::to_string(&self).unwrap();
+//         // create response and set content type
+//         ready(Ok(HttpResponse::Ok()
+//             .content_type("application/json")
+//             .body(body)))
+//     }
+// }
+
+// Implementation for User struct, functions for read/write/update/delete
+// from db
+//
```

Comment:
remove this one -- is there no like doc string equivalent in rust? like `/* ... */`? @grebneerg

Reply:
Doc comments in rust are `///`